### PR TITLE
Module feedback feature

### PR DESCRIPTION
**Pull request by dmorina**

### CODE REVIEW

File: crowdsourcing/viewsets/project.py [hunk 1]
```diff
@@ -201,8 +201,19 @@ def list_by_project(self, request, **kwargs):
     @detail_route(methods=['get'])
     def list_comments(self, request, **kwargs):
         comments = models.ModuleComment.objects.filter(module=kwargs['pk'])
-        serializer = ModuleCommentSerializer(instance=comments, many=True)
-        return Response(serializer.data, status.HTTP_200_OK)
+        serializer = ModuleCommentSerializer(instance=comments, many=True, fields=('comment', 'id',))
```
Comment (aginzberg) [inline crowdsourcing/viewsets/project.py]: typo in fields

Comment (dmorina) [inline crowdsourcing/viewsets/project.py]: what's the typo?

**Comment by aginzberg** [comment]
lgtm

**Comment by dmorina** [comment]
gracias muchacho :D

**Pull request merged**